songpal/containers.py

```diff
@@ -79,6 +79,19 @@ class PlaybackFunction:
     function = attr.ib()
 
 
+@attr.s
+class AvailableFunctions:
```

Comment:
```suggestion
class AvailablePlaybackFunctions:
```

Reply:
I was going to do that, but I matched what was done for `SupportedFunctions`. It isn't named `SupportedPlaybackFunctions`.

Reply:
I will update this and `SupportedPlaybackFunctions`.

Reply:
![Screenshot 2023-11-02 at 12 29 02 AM](https://github.com/rytilahti/python-songpal/assets/4916781/7b8ca52b-40ad-40c1-983e-5da55843a6f1)

Done.